dscache/dscache_test.go

```diff
@@ -97,7 +98,7 @@ func TestResolveRef(t *testing.T) {
 
 	dsrefspec.AssertResolverSpec(t, dsc, func(r dsref.Ref, author profile.Author, _ *oplog.Log) error {
 		builder := NewBuilder()
-		pid, err := profile.KeyIDFromPub(author.AuthorPubKey())
+		pid, err := key.IDFromPubKey(author.AuthorPubKey())
```

Comment:
Since this is explicitly now a keyID and not related to profileIDs, let's rename `pid` to `kid`.